proposed/http-message-meta.md

```diff
@@ -166,8 +161,7 @@ messages.
   there will be a certain amount of invention needed to describe HTTP message
   interfaces in PHP.
 
-5. Design Decisions
--------------------
+## 5. Design Decisions
 
 ### Message design
```

Comment:
Probably yes. Could be easier to refer to specific paragraphs. Would you like making a pull request?